instance/models/mixins/storage.py

```diff
@@ -236,70 +257,161 @@ def s3_custom_domain(self):
         """
         return "{}.s3.amazonaws.com".format(self.s3_bucket_name)
 
+    @property
+    def iam(self):
```

Comment:
Will these connections reconnect after a timeout? Should or could we test if they're still connected?

Reply:
There's a [default 60 second timeout](https://botocore.amazonaws.com/v1/documentation/api/latest/reference/config.html) and 4 retries before raising, so I think we are good here.

Reply:
Cool, thanks for verifying. That should be plenty.
Just wanted to make sure we weren't passing back stale connections.